focus-ios/focus-ios-tests/XCUITest/DragAndDropTest.swift

```diff
@@ -5,20 +5,19 @@
 import XCTest
 
 class DragAndDropTest: BaseTestCase {
-    let websiteWithSearchField = ["url": "https://developer.mozilla.org/en-US/search", "urlSearchField": "Site search..."]
+    let websiteWithSearchField = ["url": "https://developer.mozilla.org/en-US/", "urlSearchField": "Search MDN"]
 
-    func testDragElement() throws {
-        throw XCTSkip("The drag and dop opens the split view instead")
-        if iPad() {
-            let urlBarTextField = app.textFields["URLBar.urlText"]
-            loadWebPage("developer.mozilla.org/en-US/search")
+    // https://testrail.stage.mozaws.net/index.php?/cases/view/2609718
+    func testDragElement() {
+        let urlBarTextField = app.textFields["URLBar.urlText"]
+        loadWebPage(websiteWithSearchField["url"]!)
+        waitForWebPageLoad()
 
-            // Check the text in the search field before dragging and dropping the url text field
-            XCTAssertEqual(app.webViews.searchFields[websiteWithSearchField["urlSearchField"]!].placeholderValue, "Site search...")
-            // DragAndDrop the url for only one second so that the TP menu is not shown and the search box is not covered
-            urlBarTextField.press(forDuration: 1, thenDragTo: app.webViews.searchFields[websiteWithSearchField["urlSearchField"]!])
-            // Verify that the text in the search field is the same as the text in the url text field
-            XCTAssertEqual(app.webViews.searchFields[websiteWithSearchField["urlSearchField"]!].value as? String, websiteWithSearchField["url"]!)
-        }
+        // Check the text in the search field before dragging and dropping the url text field
+        waitForExistence(app.webViews.textFields[websiteWithSearchField["urlSearchField"]!])
+        // DragAndDrop the url for only one second so that the TP menu is not shown and the search box is not covered
+        urlBarTextField.firstMatch.press(forDuration: 1, thenDragTo: app.webViews.otherElements["search"].firstMatch)
```

Comment:
If I drag the URL to the `textField`, the URL does not populate the properly. Dragging to this `otherElements` does.